Lib/test/test_csv.py

```diff
@@ -454,6 +454,8 @@ def test_read_quoting(self):
                           quoting=csv.QUOTE_STRINGS)
         self._read_test(['1,@,3,@,5'], [['1', ',3,', '5']], quotechar='@')
         self._read_test(['1,\0,3,\0,5'], [['1', ',3,', '5']], quotechar='\0')
+        self._read_test(['1\\.5,\\.5,.5'], [[1.5, 0.5, 0.5]],
+                        quoting=csv.QUOTE_NONNUMERIC, escapechar='\\')
```

Comment:
Please add also test for `quoting=csv.QUOTE_STRINGS`.

Reply:
Added in 276a746cee66109d94592b5e2cfefc707e11e5f0